src/pages/individual/EditIndividualMetadata.jsx

```diff
@@ -9,6 +10,14 @@ import CustomAlert from '../../components/Alert';
 import InputRow from '../../components/fields/edit/InputRow';
 import Button from '../../components/Button';
 import StandardDialog from '../../components/StandardDialog';
+import Text from '../../components/Text';
+
+function deriveNameGuid(metadata, schemaName) {
+  const foundSchema = metadata.find(
+    schema => schema.name === schemaName,
+  );
+  return foundSchema?.nameGuid;
```

Comment:
What if there's more than one match in the .find?

Reply:
I think find takes the first matching item

Reply:
Yes, it returns the first value that satisfies the test, or, if no value does, it returns `undefined`.